_docs/6_BlockArchives/docker-compose.yml

```diff
@@ -437,7 +437,7 @@ services:
     restart: unless-stopped
     environment:
       DEBUG: 1
-      SOURCE_CHAIN_RPC: wss://nodle-parachain.api.onfinality.io/public-ws
+      SOURCE_CHAIN_RPC: wss://nodle-parachain.api.onfinality.io/public-ws?apikey={ONFINALITY_API_KEY}
```

Comment:
Are you sure it is public endpoint with token? Previously it was `/ws` endpoint instead.
Also it is a bit non-obvious that you'd need to set an environment variable before running `docker-compose` might be easy to forget.

Reply:
no, it's `public` or `public-ws`, you check above.

if one forgets to set env variable, then it will just use the public endpoint - nothing will break

Reply:
I mean I don't disagree that it may work, but I'm fairly confident that API does nothing. How did you verify it actually does anything and why did you not use `/ws` endpoint as shown in the OnFinality dashboard?

Reply:
I always used `public-ws` and I know that without apiKey it disconnects more often. Why do you think API does not do anything?

Reply:
they are not replying... I can just put `/ws` everywhere

Reply:
Yes, let's do that, please